frontend/admin/src/authProvider.tsx

```diff
@@ -1,28 +1,53 @@
+import { initializeApp } from 'firebase/app'
+import { getAuth, signInWithEmailAndPassword } from 'firebase/auth'
+
 import { User, UserRole } from './models'
 
-const login: (params: { username: string }) => Promise<void> = ({
-  username,
-}) => {
-  const isSuperadmin = username === 'superadmin'
-  const user = isSuperadmin
-    ? {
-        email: username,
-        name: 'Super Admin',
-        tenantId: '',
-        role: UserRole.SUPER_ADMIN,
-      }
-    : {
-        email: username,
-        name: 'Tenant Admin',
-        tenantId: '1',
-        role: UserRole.TENANT_ADMIN,
-      }
-  localStorage.setItem('user', JSON.stringify(user))
-  return Promise.resolve()
+const firebaseConfig = {
+  apiKey: 'AIzaSyAJ_GxyUXUkja9DCHXVTbH9Jhje5bsfv9s',
+  authDomain: 'coworkreservation.firebaseapp.com',
+  projectId: 'coworkreservation',
+  storageBucket: 'coworkreservation.appspot.com',
+  messagingSenderId: '182196777623',
+  appId: '1:182196777623:web:c9d18ac242055ba54f2fd2',
+  measurementId: 'G-LRR9NTQ779',
 }
 
-const logout: () => Promise<void> = () => {
+initializeApp(firebaseConfig)
+const auth = getAuth()
+
+const login: (params: { username: string; password: string }) => Promise<void> =
+  async ({ username, password }) => {
+    try {
+      await signInWithEmailAndPassword(auth, username, password).then(
+        (userCredential) => {
+          const isSuperadmin =
+            userCredential.user.email?.split('@')[0] === 'superadmin'
+          const user = isSuperadmin
```

Comment:
I know this will be replaced, but I will write this to make sure you know how it works. The user role will be contained in the custom claims so you won't need to use local storage. You can get the user from the firebase SDK calls.